components/coding/Expandable.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+
+import { PlusCircleIcon } from "@heroicons/react/solid";
+import { MinusCircleIcon } from "@heroicons/react/solid";
+
+interface ExpandableProps {
+  open?: boolean;
+  title: string;
+}
+
+const ExpandableContainer: React.FC<ExpandableProps> = ({
+  open,
+  children,
```

Comment:
I noticed that `children` isn't defined in the expandable props but Typescript knows it's type because it's a property of `React.ReactNode`. 

I probably would have explicitly defined a child property of type React.FC in the ExpandableProps - which I confirmed works but actually seems unnecessary if children is already part of React.FC. So I think what you're doing is right but wanted to point it out to make sure you understand what's going on.

cc @vithushan19 if you have any extra context for us here